licensedb/internal/processors/html2text.go

```diff
@@ -0,0 +1,60 @@
+package processors
+
+import (
+	"bytes"
+	stdhtml "html"
+	"strings"
+
+	"golang.org/x/net/html"
+)
+
+var (
+	skipped = map[string]bool{
+		"head":   true,
+		"script": true,
+		"style":  true,
+		"object": true,
+	}
+	regexBytes = []byte("#$%*/\\\\|><~`=!?.,:;\"'\\])}-")
+)
+
+func isHeader(b []byte) bool {
+	if len(b) < 2 {
+		return false
+	}
+	return b[0] == 'h' && b[1] >= '2' && b[1] <= '6'
+}
+
+func isRegex(b byte) bool { return bytes.ContainsRune(regexBytes, rune(b)) }
+
+// HTML converts HTML to plain text. E.g. it rips all the tags.
+func HTML(src []byte) []byte {
```

Comment:
The name is bit generic. How about `ExtractPlainTextFromHTML` or `HTMLToPlainText` or `ExtractTextFromHTML` or ...